src/Exception/InvalidServiceConfigException.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-di for the canonical source repository
+ * @copyright Copyright (c) 2019 Zend Technologies USA Inc. (https://www.zend.com)
+ * @license   https://github.com/zendframework/zend-di/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Zend\Di\Exception;
+
+use Psr\Container\ContainerExceptionInterface;
+
+class InvalidServiceConfigException extends LogicException implements ContainerExceptionInterface
```

Comment:
not sure if LogicException as a parent is the correct one

Maybe better InvalidArgumentException?

Reply:
InvalidArgumentException actually is a LogicException.
But in this case I'd argue that this is not cased by a function argument but a (service)configuration value. This could also be seen a `RuntimeException`.

I chose LogicException since it's the developer's job to configure the services correctly, and if they don't provide a namespace-string, they made a logical mistake.

> https://www.php.net/manual/en/class.logicexception.php
> Exception that represents error in the program logic. This kind of exception should lead directly to a fix in your code. 

But I'm happy if you provide a better fit.

Reply:
Ok, sounds good to me.

I just wonder if we should have then another exception before if the namespace is provided but is not null and not a string? Maybe we shouldn't ignore invalid value. What do you think?

Reply:
Isn't this already the case or am I missing something? :thinking: